scripts/compatibility.lua

```diff
@@ -29,6 +29,32 @@ function compatibility.nuclear_fuel()
   end
 end
 
+function compatibility.atomic_overhaul()       --Added This
```

Comment:
By comparing commits, it is obvious that this function is added.

```suggestion
function compatibility.atomic_overhaul()
```